civihr_employee_portal/js/ta-documents-app.js

```diff
@@ -46,17 +44,14 @@
          * @param {string} mode
          */
         function modalDocument (id, role, mode) {
-          $rootScope.$broadcast('ct-spinner-show', 'document-' + id);
-          vm.loadingModalData = true;
-
           DocumentService.get({ id: id })
             .then(function(data) {
               if (!data) {
                 throw new Error('Requested Document is not available');
               }
-
+              vm.openModalDocument(data[0], role, mode);
               vm.cacheContacts(data).then(function () {
-                vm.openModalDocument(data[0], role, mode);
+                $rootScope.isLoading = false;
               });
             })
             .catch(function(reason) {
```

Comment:
the fact that there is no space between `function` and `(reason)` should have thrown an error in the linter, are you sure you have the git hook configured correctly

Reply:
I had permission issue in executing the pre-commit hook. Thanks for pointing it out! fixed!